dbpedia_Classical_musicians_by_nationality/package.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
Haha, i guess that was created from the auto fill "npm init -y". But you're right, I'll remove the package file!